### ISSUE TITLE
Error S6001 on Linux Fedora

### ISSUE THREAD

**Issue by james-westwood** [Mastermindzh/tidal-hifi#353]
I have googled around for this error code but found nothing relevant. 

Here's a screenshot of the error message that appears at the bottom of the player. 

![image](https://github.com/Mastermindzh/tidal-hifi/assets/67740306/115ff171-9f95-4824-8898-5f433b315ca6)

The whole interface seems to be loading correctly, but I have not been able to get it to play any music. 

Happy to give any other specs of my machine or set-up if it's relevant. Please let me know what you need.

**Comment by Mastermindzh** [comment]
Hey,

This seems to happen for some people at times and is usually fixed by deleting the cache of TIDAL.

Try:

`rm -rf ~/.config/tidal-hifi`

(Or wherever the config dir is located)

**Comment by james-westwood** [comment]
Hi, unfortunately this did not work. The config is indeed stored at the location you suggested, and clearing meant I had to log in again. 

However when I tried to play a track the same error occurred.

**Comment by james-westwood** [comment]
Hi again. I couldn't solve this issue so I uninstalled the `dnf` version and installed again via flatpack, as I noticed [this other user](https://github.com/Mastermindzh/tidal-hifi/issues/347#issue-2137603774) had solved their problem by using flatpack. 

This solved two problems for me:
- The S6001 error I was getting
- The icon was missing, but now I have it. 

Everything seems to work so far. 

Thanks!
